type_templates/vrm.js

```diff
@@ -146,6 +146,21 @@ export default e => {
 
   })());
 
+
+  useFrame(({ timestamp, timeDiff }) => {
+    if (app.avatar){

```

Comment:
I suppose this conflicts with the NPC case.

Would need to only do this if the local player avatar is this one.